doc/en/py27-py34-deprecation.rst

```diff
@@ -1,31 +1,96 @@
-Python 2.7 and 3.4 support plan
-===============================
+Python 2.7 and 3.4 support
+==========================
 
-Python 2.7 EOL is fast approaching, with
-upstream support `ending in 2020 <https://legacy.python.org/dev/peps/pep-0373/#id4>`__.
-Python 3.4's last release is scheduled for
-`March 2019 <https://www.python.org/dev/peps/pep-0429/#release-schedule>`__. pytest is one of
-the participating projects of the https://python3statement.org.
+It is demanding on the maintainers of an open source project to support many Python versions, as
+there's extra cost of keeping code compatible between all versions, while holding back on
+features only made possible on newer Python versions.
 
-The **pytest 4.6** series is the last to support Python 2.7 and 3.4, and was released in
-**June 2019**. **pytest 5.0** and onwards will support only Python 3.5+.
+In case of the Python 2 and 3, the difference between the languages makes it even more prominent,
+because many new Python 3 features cannot be used in a Python 2/3 compatible code base.
+
+Python 2.7 EOL has been reached `in 2020 <https://legacy.python.org/dev/peps/pep-0373/#id4>`__, with
+the last release planned for mid-April, 2020.
+
+Python 3.4 EOL has been reached `in 2019 <https://www.python.org/dev/peps/pep-0429/#release-schedule>`__, with the last release made in March, 2019 .
+
+For those reasons, in Jun 2019, it was decided that **pytest 4.6** series will be the last to support Python 2.7 and 3.4.
+
+What this means for general users
+---------------------------------
 
 Thanks to the `python_requires`_ ``setuptools`` option,
 Python 2.7 and Python 3.4 users using a modern ``pip`` version
-will install the last pytest ``4.6`` version automatically even if ``5.0`` or later
+will install the last pytest ``4.6`` version automatically even if ``5.0`` or later versions
```

Comment:
No need for code formatting for version numbers, it's not done for "Python 2.7" or "**pytest 4.6**" or "4.6.X" in some places. 

```suggestion
will install the last pytest 4.6 version automatically even if 5.0 or later versions
```